src/components/charts/histogram/index.jsx

```diff
@@ -374,69 +375,73 @@ class Histogram extends React.PureComponent {
-          className={svgClasses}
-          preserveAspectRatio="xMidYMid meet"
-          viewBox={`0 0 ${width} ${height}`}
-          ref={this.svgEl}
-        >
-          {xScale && yScale && multiple && (
-            <>
-              {data.map((dataset, i) => {
-                const groupName = groupNames[i];
-                const key = groupName + i;
-
-                return (
-                  <Bars
-                    barClasses={groupName}
-                    colorClass={`color-set-${i + 1}`}
-                    key={key}
-                    data={dataset}
-                    selectedData={selectedData}
-                    hoveredData={hoveredData}
-                    offsetTop={offsetTop}
-                    xScale={xScale[i]}
-                    yScale={yScale[i]}
-                    graphHeight={height}
-                    padding={padding}
-                  />
-                );
-              })}
-            </>
+        <div className="svg-container histogram-container">
+          {loading && (
+            <CircularProgress
+              id="graph-loading-progress"
+              className="chart-loader"
+              scale={3}
+            />
           )}
-          {xScale && yScale && !multiple && (
-            <>
-              <Bars
-                data={data}
-                selectedData={selectedData}
-                hoveredData={hoveredData}
-                offsetTop={offsetTop}
-                xScale={xScale}
-                yScale={yScale}
-                graphHeight={height}
-                padding={padding}
-              />
-              {/* {meanData && valueAccessor && (
+          <Tooltip
+            key="tooltip"
+            graph="histogram"
+            data={selectedData || hoveredData}
+            posX={tooltipPosX}
+            posY={tooltipPosY}
+            show={showTooltip}
+            accessors={tooltipAccessors}
+            units={tooltipUnits}
+            labels={tooltipLabels}
+          />
+          <svg
+            key={valueAccessor}
+            className={svgClasses}
+            preserveAspectRatio="xMidYMid meet"
+            viewBox={`0 0 ${width} ${height}`}
+            ref={this.svgEl}
+          >
+            {xScale && yScale && multiple && (
+              <>
+                {data.map((dataset, i) => {
+                  const groupName = groupNames[i];
+                  const key = groupName + i;
+
+                  return (
+                    <Bars
+                      barClasses={groupName}
+                      colorClass={`color-set-${i + 1}`}
+                      key={key}
+                      data={dataset}
+                      selectedData={selectedData}
+                      hoveredData={hoveredData}
+                      offsetTop={offsetTop}
+                      xScale={xScale[i]}
+                      yScale={yScale[i]}
+                      graphHeight={height}
+                      padding={padding}
```

Comment:
You have a lot of common `key: value` pairs here.  Would it be okay practice to spread the common pairs? like:
`
{...{
  selectedData,
  hoveredData,
  offsetTop,
  padding,
}}
`